ImprovedAnvil/lib/effects/02_create_new.tph

```diff
@@ -0,0 +1,5 @@
+/// new effects that were created by copying and patching the existing ones ///

```

Comment:
this file won't fire unless you include it specifically into setup.tph (I think?)